### DISCUSSION TITLE
where to get older development versions?

### DISCUSSION THREAD

**Discussion by pr-apes** [speedata/publisher#498] [Q&A] [answered]
@pgundlach,

a tiny XML layout that I use on almost a daily basis is misbehaving (or behaving it differently as before).

I'm not aware of having changed anything on my part. I would like to know whether this is a regression in Publisher (to give an accurate bug report, in case Publisher is failing here).

The previous description is only context for the question:

Is there a way to access older development versions?

https://download.speedata.de/ provides access to older versions, but only to stable releases.

Many thanks for your help.

**Comment by pgundlach** [comment, marked as answer]
There are no older (development) versions. If you know know a version number, it would possible to re-generate a ZIP file from the source.

**Reply by pr-apes** [reply]
Well, I will have to provide a minimal sample (tested against current latest for Publisher).
